frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -865,6 +865,9 @@ private void prepareDetailView(RenderRequest request, RenderResponse response) t
                         VULNERABILITY_RATINGS_EDITABLE,
                         PermissionUtils.makePermission(project, user).isActionAllowed(RequestedAction.WRITE));
 
+                request.setAttribute(
+                        ATTCHMENT_USAGE_EDITABLE,
+                        PermissionUtils.makePermission(project, user).isActionAllowed(RequestedAction.ATTACHMENTS));
```

Comment:
The update action that is used when hitting the "Save Usages" button is org.eclipse.sw360.portal.portlets.projects.ProjectPortlet.saveAttachmentUsages(ResourceRequest, ResourceResponse) which checks only the WRITE permission. Do we really want to check the ATTACHMENTS permission here for client side en-/disabling?

Reply:
Currently, ATTACHMENT and WRITE serves the same purpose for project level permission. But as suggested, I will rename "VULNERABILITY_RATINGS_EDITABLE" to a generic one "WRITE_ACCESS_USERS" and make the corresponding changes where ever applicable and use the same variable for attachment usages as well.